pkg/generators/snippets/deepcopy.go

```diff
@@ -9,44 +9,42 @@ import (
 )
 
 func GenerateDeepCopyInto(w io.Writer, ctx *generator.Context, t *types.Type) error {
-	if ctx == nil || t == nil {
```

Comment:
Moving `ctx == nil` checks into `writeSnippetWithArgs` to avoid having to do this check every time.